bundles/org.palladiosimulator.probeframework/src/org/palladiosimulator/probeframework/calculator/ExtensibleCalculatorFactoryDelegatingFactory.java

```diff
@@ -0,0 +1,63 @@
+package org.palladiosimulator.probeframework.calculator;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.Objects;
+
+import org.palladiosimulator.edp2.models.measuringpoint.MeasuringPoint;
+import org.palladiosimulator.metricspec.MetricDescription;
+import org.palladiosimulator.probeframework.probes.ProbeConfiguration;
+
+/**
+ * This factory allows to use specialized factories for Calculators of dedicated
+ * metrics. Calculator factories can be registered based on the id of the
+ * metric. In case no dedicated factory is registered, a fallback factory can be
+ * provided.
+ * 
+ * By default the factory uses the <code>CalculatorFactorRegistry</code> to look
```

Comment:
I cannot find any usage of the CalculatorFactor(y)Registry in the code. It is some sort of assumption that the registry result is passed as first parameter in the constructor, right?

Reply:
Fixed